dff/core/engine/conditions.py

```diff
@@ -111,12 +111,13 @@ def aggregate_condition_handler(ctx: Context, actor: Actor, *args, **kwargs) ->
 
 @validate_arguments
 def any(cond_seq: list, *args, **kwargs) -> Callable:
```

Comment:
Should we add Callable signature (☝️): `Callable[[Context, Actor, Any, Any], bool]` (AFAIK) here.